templates/designatebackendbind9/config/named.conf

```diff
@@ -0,0 +1,5 @@
+include "/etc/named/rndc.key";
+include "/etc/named/rndc.conf";
+include "/etc/named/options.conf";
+include "/etc/named.rfc1912.zones";
+include "/etc/named/logging.conf";
```

Comment:
In the old templates/designate/config/named.conf we also had 
include "/etc/named.root.key";

Reply:
I'll double check

Reply:
We do not have that in the tripleo config so I'm leaving out.